Revise TAC and SCIR election timeline (#525)
ossf/tac: elections/tac-and-scir-election-process.md

## Appointment Process

> The Governing Board will appoint three people pulling from the list of TAC and SCIR Nominees excluding those who got elected and those they nominate. The Governing Board will be provided with the results of the election from the community so that they can ensure appropriate diversity of the TAC during their vote.

Reviewer: ```suggestion
The Governing Board will appoint three (3) seats on the Technical Advisory Council. These individuals will be selected from the following nominee categories:

- TAC and SCIR nominees for the current election who were _not_ elected
- individuals from [OpenSSF member organizations](https://openssf.org/about/members/) that are nominated by Governing Board representatives 

The Governing Board will be provided with the results of the election from the community so that they can ensure appropriate diversity of the TAC during their vote.
```

Made some adjustments here so that the exclusion language reads more clearly.

Reviewer: > ensure appropriate diversity

What does this mean more precisely?

| October 23 - October 24 | Validation Period of nominee submissions
| October 27 | Slate of respective nominees and ballot or link to an electronic voting system sent to registered voters.
| November 9 | The voting period ends at 11:59 pm PDT
| November 11 | The election winners announced

> **[Community Election Self-Nomination Form](https://docs.google.com/forms/d/e/1FAIpQLScNzK_8bApE7mIvTZpFuP00Lmc6syQkPGe-JYGekp37DSswtQ/viewform?usp=dialog)**

Reviewer: ```suggestion
**[Community Election Self-Nomination Form](https://docs.google.com/forms/d/e/1FAIpQLScNzK_8bApE7mIvTZpFuP00Lmc6syQkPGe-JYGekp37DSswtQ/viewform)**
```

> **[SCIR Self-Nomination Form](https://docs.google.com/forms/d/e/1FAIpQLSe-9A1I0LM2BJA5_TOGADxUapvAdzgZHsF7IwnURqEZJJigaQ/viewform?usp=dialog)**

Reviewer: ```suggestion
**[SCIR Self-Nomination Form](https://docs.google.com/forms/d/e/1FAIpQLSe-9A1I0LM2BJA5_TOGADxUapvAdzgZHsF7IwnURqEZJJigaQ/viewform)**
```

**Governing Board Appointed Seat Timeline**

| Date | Action |
| :--- | :--- |
| November 10 | Call for nominations; notice given to GB Members and include results from the community election.
| November 16 | End nomination period, 11:59 PM PST.
| November 17 | The slate of respective nominees and ballot or link to an electronic voting system sent to the governing board primary voting representatives.
| November 30 | The voting period ends at 11:59 PM PST.
| December 2 | The GB-appointed TAC representatives announced.

 **New members seated: January 1, 2026**

## Voter Registration form

> The electorate can request a ballot by filling out this **[Google Form](https://docs.google.com/forms/d/e/1FAIpQLScpX1Fdlvcj_9JwYVzQlDQrgHfXWSd8v9ruc2ZKmrfq4Yf5lQ/viewform?usp=dialog).**

Reviewer: ```suggestion
The electorate can request a ballot by filling out this **[Google Form](https://docs.google.com/forms/d/e/1FAIpQLScpX1Fdlvcj_9JwYVzQlDQrgHfXWSd8v9ruc2ZKmrfq4Yf5lQ/viewform).**
```
